docker-compose.yml

```diff
@@ -34,4 +34,4 @@ services:
 
 volumes:
   dbdata:
-  minio_data:
\ No newline at end of file
+  minio_data:
```

Comment:
Can you configure your editor to stop doing that? ^^

Reply:
Newlines at EOF?

Reply:
yep

Reply:
That's something I mostly do manually. But yeah. I can stop.

What's the reasoning behind not having newlines at the end of the files for some file types? And what types require newlines and which don't?

Reply:
Ah ok. I don't know why it's preferred not to have those newlines, but I do know that it's nice to have a consistent default for this all over the repository. My editor is configured to remove those newlines, and I think most of the people in our team handle it the same way.
I'd just like to get of the extra noise in those merge requests :wink:

Reply:
Ok